pkg/operation/seed/scheduler/gardener_kube_scheduler.go

```diff
@@ -0,0 +1,438 @@
+// Copyright (c) 2020 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
```

Comment:
We already have a package for the kube-scheduler, https://github.com/gardener/gardener/tree/master/pkg/operation/botanist/controlplane/kubescheduler. Can't we rather extend it to work for both use-cases?

Reply:
There are differences - RBAC, serviceaccounts etc. This was my first idea, but then the code became very hard to test - multiple branches and code paths.